packages/cli/src/lib/local/run-foreman.js

```diff
@@ -0,0 +1,286 @@
+/* eslint-disable block-scoped-var */
```

Comment:
Can we add some comments to this to explain a little bit of what you said in the PR summary?